src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let sum = days * 40;
+
+  if (sum < 120) {
+    return sum;
+  }
+
+  if (sum >= 280) {
+    sum -= 50;
+
+    return sum;
+  }
```

Comment:
The condition `sum >= 280` is using a hardcoded value which is not clear. Use a named constant to represent the threshold for the long-term rental discount, and another constant for the actual discount amount.